EIPS/eip-831.md

```diff
@@ -20,27 +18,25 @@ URIs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages prov
 
 ### Syntax
 
-Ethereum URIs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+Ethereum URIs contain "ethereum" or "eth" in their schema (protocol) part and are constructed as follows:
 
-    request                 = "ethereum" ":" [ prefix "-" ] payload
+    request                 = "eth" [ "ereum" ] ":" [ prefix "-" ] payload
     prefix                  = STRING
     payload                 = STRING
 
 ### Semantics
 
-`prefix` is optional and defines the use-case for this URI. If no prefix is given: "pay-" is assumed to be concise and ensure backward compatibility to ERC-67. When the prefix is omitted, the payload must start with `0x`. Also prefixes must not start with `0x`. So starting with `0x` can be used as a clear signal that there is no prefix.
-
-`payload` is mandatory and the content depends on the prefix. Structuring of the content is defined in the ERC for the specific use-case and not in the scope of this document. One example is ERC-681 for the pay- prefix.
+`prefix` is optional and defines the use-case for this URI. If no prefix is given: "pay-" is assumed to be concise and ensure backward compatibility to [EIP-67](https://github.com/ethereum/EIPs/issues/67). When the prefix is omitted, the payload must start with `0x`. Also prefixes must not start with `0x`. So starting with `0x` can be used as a clear signal that there is no prefix.
```

Comment:
There is no EIP-67, so you cannot link to EIP-67.  Recommend just removing this comment about compatibility.  You can't be backward incompatible with a thing that doesn't exist.  😄

Reply:
I really want to leave this in as it gives important context.

Reply:
```suggestion
`prefix` is optional and defines the use-case for this URI. If no prefix is given: "pay-" is assumed to be concise and ensure backward compatibility with other standards. When the prefix is omitted, the payload must start with `0x`. Also prefixes must not start with `0x`. So starting with `0x` can be used as a clear signal that there is no prefix.
```
Fixed.

Reply:
What other standards are being referred to by "with other standards"?  If there are no standards then this feels like a vacuous statement.

Perhaps instead, "backward compatibility with existing contracts in the ecosystem" to make it clear that there are no existing standards but there does exist contracts in the wild that there is a desire to be compatible with?

Reply:
> What other standards are being referred to by "with other standards"? If there are no standards then this feels like a vacuous statement.

There are other standards out there that aren't EIPs, or even official standards. "EIP-223" (not an EIP) is a good example of this.

Reply:
At best, that is a defacto standard which I personally wouldn't call a standard.